contracts/wallet.sol

```diff
@@ -453,6 +452,7 @@ contract Wallet is Vault {
 
     using SafeMath for uint256;
 
+    string constant public WALLET_VERSION = "1.5.8";
```

Comment:
I would put the v1.5.8, as that is what we are doing in the repo. The tag is "v1.5.8"